packages/workspace/src/browser/workspace-frontend-contribution.ts

```diff
@@ -141,6 +141,12 @@ export class WorkspaceFrontendContribution implements CommandContribution, Keybi
     }
 
     registerKeybindings(keybindings: KeybindingRegistry): void {
+        if (this.isElectron()) {
+            keybindings.registerKeybinding({
```

Comment:
wouldn't it make sense to register the command for browser as well? In that case use `shift` in addition?

Reply:
Just figured that will create a private browser in chrome. So maybe use `alt`.

Reply:
`alt+n` could work. Thanks for suggestion!